scss/_patterns_switch.scss

```
@@ -29,27 +35,27 @@ $knob-size: $sp-unit * 3;
   }
 
   .p-switch__slider {
-    @extend %vf-has-round-corners;
-
-    background: linear-gradient(to right, $color-link 50%, $color-mid-light 50%);
+    background: $color-mid-light;
+    border-radius: $knob-size * 1.5;
```

Should the border radius here just equal the size of the handle?
```suggestion
    border-radius: $knob-size;
```